libraries/GCS_MAVLink/GCS.h

```diff
@@ -746,6 +747,9 @@ class GCS_MAVLINK
     mavlink_message_t _channel_buffer;
     mavlink_status_t _channel_status;
 
+    // for mavlink radio
+    mavlink_radio_t _mavlink_radio_packet;
```

Comment:
better to make this a pointer and allocate on first receiver so we don't pay for it on every connection

Reply:
this is actually not needed for this PR but a leftover. Will be needed for a potential follow PR doing link statistics. Field removed.